RAMAnimatedTabBarController/RAMAnimatedTabBarController.swift

```diff
@@ -69,21 +69,21 @@ class RAMAnimatedTabBarController: UITabBarController {
     func createCustomIcons(containers : NSDictionary) {
 
         if let items = tabBar.items {
-            let itemsCount = tabBar.items!.count as Int - 1
-            var index = 0
-            for item in self.tabBar.items as! [RAMAnimatedTabBarItem] {
+            let itemsCount = items.count as Int - 1
+            
+            for (index, item) in enumerate(items as! [RAMAnimatedTabBarItem]) {
 
                 assert(item.image != nil, "add image icon in UITabBarItem")
```

Comment:
If we remove this error message, this project will crash because we don't item.image nil check.

It is more safe to add nil check.